assert/assert-core/src/test/java/com/mastercard/test/flow/assrt/ReportCustomizerTest.java

```diff
@@ -0,0 +1,80 @@
+package com.mastercard.test.flow.assrt;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I always try to avoid wildcard imports - they make it more difficult to trace classes back to their definition.
You can configure your IDE to avoid these.